README.md

```diff
@@ -89,12 +89,20 @@ docker run -it --env-file ./.env -p 5000:5000 todo-app:prod
 - To run all tests in a directory, run `poetry run pytest <<directory>>`
 - To run an indiviual test, run the command `pytest -k <<test_name>>` from your terminal or run from vscode > testing
 - In Docker:
--- `docker build --target tests --tag todo-app:tests .`
--- `docker run --env-file ./.env.test todo-app:tests`
+    - `docker build --target tests --tag todo-app:tests .`
+    - `docker run --env-file ./.env.test todo-app:tests`
 
 ## Provisioning a VM from an Ansible Control Node
 
 - Set up SSH from the Control to Host(s) ansible node(s)
 - Copy/Recreate the files in the ansible folder onto the control node
 - Update the necessary values in `.env.j2` and `ansible-inventory`
-- Run `ansible-playbook ansible-playbook.yml -i ansible-inventory` to provision
\ No newline at end of file
+- Run `ansible-playbook ansible-playbook.yml -i ansible-inventory` to provision
+
+## Azure Hosted App
+
+- Azure container image is hosted at https://hub.docker.com/repository/docker/danielharedevops/todo-app/general
+- To build and push use:
+    - `docker build --target production --tag danielharedevops/todo-app .`
+    - `docker push danielharedevops/todo-app`
+- To push to Azure make a POST request to the webhook link in the Deployment Center of Azure.
```

Comment:
Thanks for the clear docs, I'm also a fan of explicitly including the application URL (<https://todo-app-service.azurewebsites.net/>) - it may sound silly but I've definitely joined teams in the past and then struggled to work out where the dev & prod environments were even deployed!

Reply:
Yeah makes sense, thanks